src/unity.yml

```diff
@@ -154,12 +158,16 @@ components:
       $ref: './unity/schemas/Me.yml'
     OrgUsers:
       $ref: './unity/schemas/OrgUsers.yml'
+    OperatorOrganization:
+      $ref: './unity/schemas/OperatorOrganization.yml'
+    OperatorOrganizations:
+      $ref: './unity/schemas/OperatorOrganizations.yml'
+    RelatedAccount:
```

Comment:
Do you think it's worth renaming `RelatedAccount` to `OperatorRelatedAccount` as well just so it's clear what it belongs to?

Reply:
In theory, `RelatedAccount` can be used for more than just operator, so I am not sure it needs to be renamed

Reply:
I feel like we could say the same thing for the schemas that are already prefixed with `Operator`, but that all of them will very likely not be used outside of the operator API, so that's why I suggested it, but I'm fine with not renaming it too.